### ISSUE TITLE
improve content accessibility by expanding word contractions

### ISSUE THREAD

**Issue by serahkiburu** [carpentries/carpentries.org-archive#993]
Word contractions might make readability and comprehension more difficult for some, and it is best to avoid these. [This post provides more context around this](https://medium.com/@joanne.schofield/using-contractions-could-be-making-your-writing-inaccessible-4034bb3cd76a). 

We invite you to help us make our content more accessible by [creating Pull Requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) in this repository to  word contractions where relevant. If in doubt, please ask for clarification under this issue.

(not required but nice to have)Where possible, you can also [mention this issue in your Pull Request](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) to enable us to keep track of accessibility improvements made on this specific issue.

Thank you for your contributions.

**Comment by maneesha** [comment]
I just put in another PR (#1155) to address some more contractions but would like to learn more about how contractions interfere with accessibility and comprehension.  The blog post cited in opening this issue doesn't refer to any research, and doesn't follow this advice itself (i.e. there are several contractions in that blog post).  There are also several contractions in the guide that the blog post cites. 

Additionally, in expanding contractions, I've found that I may be changing the voice of the text, especially for texts written in the first person.  

For all these reasons, I'm skeptical and reluctant to universally expand all contractions.  This is why I haven't done a lot of work to expand them.  I'm curious to hear about research that tells us about how contractions impact comprehension.  Leaving this issue open for people to share what they know and/or have experienced.

(completely self-aware of all the contractions in my own text above)

**Comment by kaylendwyer** [comment]
@maneesha I spent some time trying to verify whether contractions decrease readability. I went through guidelines on plain language from various organizations and tried to find studies on readability. Unfortunately, there is no final consensus on using contractions. Plain language guidelines rarely cite research to back up their claims, and I had trouble tracking down their sources. 

**Overall:** It seems that contractions make content more readable for native speakers of English but less readable for people who are bilingual or speak English as a second language. Since the SWCs are transnational, it seems like expanding contractions might be the best practice.

**Recommend Contractions:** 
[The Document Design Project’s “Guidelines for Document Designers,”](https://redish.net/wp-content/uploads/Guidelines_for_Document_Designers_2014.pdf) may be the most comprehensive evidence-based guide for writing in plain language. The guidelines don’t address contractions, but they do use them throughout the entire document.

[PlainLanguage.gov](https://www.plainlanguage.gov/guidelines/conversational/use-contractions/) – suggests using contractions on the basis that it is less formal and more natural (“write like you talk”).

[Google developer documentation guidelines](https://developers.google.com/style/contractions)  -- recommends using negation contractions for clarity
> Negation contractions
> In particular, it's fine to use -n't contractions, such as isn't, don't, and can't.
> One reason that such contractions are useful is that it's sometimes easy for a reader to miss the word not, whereas it's harder to misread don't as do.

[Microsoft’s style guide](https://docs.microsoft.com/en-us/style-guide/word-choice/use-contractions) -- states “don’t mix contractions,” such as using _can’t_ and _cannot._

Chicago Manual of Style and MLA

**Discourage Contractions:**
[Gov.UK’s guidelines](https://www.gov.uk/guidance/content-design/writing-for-gov-uk#contractions) -- discourage using negative contractions. The “research” they allude to may have been their own usability testing.

[Simple English Wikipedia](https://simple.wikipedia.org/wiki/Wikipedia:Manual_of_Style)

**Comment by maneesha** [comment]
Assigning this to @brynnelliott - not sure what your workflow or workload is like, so prioritize as you see fit

**Comment by maneesha** [comment]
Closed based on comment from @brynnelliott  here (https://github.com/carpentries/docs.carpentries.org/issues/702)
